.github/workflows/linux-build-base.yml

```diff
@@ -142,6 +144,53 @@ jobs:
 
           export CLASSPATH=`/usr/local/hadoop/bin/hdfs classpath --glob`
           ctest -j 8 --label-exclude cuda_driver --output-on-failure --no-tests=error
+           - name: Upload writer fuzzer
+
+      - name: Check cuDF Changes
+        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
+        id: changes
+        if: ${{ ! inputs.use-clang }}
+        with:
+          filters: |
+            cudf:
+              - 'velox/exprimental/cudf/**'
```

Comment:
Do you also detect this file? https://github.com/facebookincubator/velox/blob/main/CMake/resolve_dependency_modules/cudf.cmake

Reply:
I added this.